### PR TITLE
feat: capture secretstore resources in support bundle

### PR DESCRIPTION
**Pull request by Elsie4ever**
add --svc secretstore to capture runtime resources, CRs are mainly under aio namespace and rest of the runtime resources are under azure-secret-store namespace

image

image

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

**Comment by Elsie4ever** [comment]
IT runs
image

File: azext_edge/edge/common.py [hunk 1]
```diff
@@ -154,6 +154,7 @@ class OpsServiceType(ListableEnum):
     dataflow = "dataflow"
     schemaregistry = "schemaregistry"
     arccontainerstorage = "acs"
+    secretsynccontroller = "ssc"
```
Comment (digimaun) [inline azext_edge/edge/common.py:157]: lets call it secretstore, including outward representation

**Comment by Elsie4ever** [comment]
latest IT runs
image

File: azext_edge/tests/edge/support/create_bundle_int/helpers.py [hunk 2]
```diff
@@ -238,7 +240,9 @@ def get_file_map(
     mq_traces: bool = False,
 ) -> Dict[str, Dict[str, List[Dict[str, str]]]]:
     # Remove all files that will not be checked
-    arc_namespace, aio_namespace, acs_namespace, c_namespace = process_top_levels(walk_result, ops_service)
+    arc_namespace, aio_namespace, acs_namespace, ssc_namespace, c_namespace = process_top_levels(
```
Comment (vilit1) [inline azext_edge/tests/edge/support/create_bundle_int/helpers.py:243]: tbh this may need to be a dict at some point - tis too large for a tuple

Comment (vilit1) [reply azext_edge/tests/edge/support/create_bundle_int/helpers.py:243]: if you dont want to do this - put a todo and I will deal with it later

Comment (Elsie4ever) [reply azext_edge/tests/edge/support/create_bundle_int/helpers.py]: updated, please feel free to take another look

**Comment by Elsie4ever** [comment]
latest IT
image

**Review by c-ryan-k — APPROVED** [review]
Looks good to me, @vilit1 could you take one more pass with your support bundle eyes to make sure everything looks good?

**Review by vilit1 — APPROVED** [review]
lgtm
image

**Pull request merged**